builtin/providers/aws/resource_aws_elb_attachment.go

```diff
@@ -0,0 +1,124 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsElbAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsElbAttachmentCreate,
+		Read:   resourceAwsElbAttachmentRead,
+		Delete: resourceAwsElbAttachmentDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Yes, thanks for catching that, it was left from an earlier attempt. 

I think importing would have to rely on the ELB importing them as a single list, which I think is probably fine. Either way we don't need Importer here, and can sort it out later if there are other options.